examples/acm-terraform-blog/part1/README.md

```diff
@@ -0,0 +1,77 @@
+# Enable ACM features with Terraform
+
+## Part 1
+
+1. Clone this repo
+1. Set variables that will be used in multiple commands:
+
+    ```bash
+    FOLDER_ID = [FOLDER]
+    BILLING_ACCOUNT = [BILLING_ACCOUNT]
+    PROJECT_ID = [PROJECT_ID]
+    ```
+
+1. Create project:
+
+    ```bash
+    gcloud auth login
+    gcloud projects create $PROJECT_ID --name=$PROJECT_ID --folder=$FOLDER_ID
+    gcloud alpha billing projects link $PROJECT_ID --billing-account $BILLING_ACCOUNT
+    gcloud config set project $PROJECT_ID
+    ```
+
+1. Enable the Google Cloud APIs that will be used for this example and also enable the new configuration management feature for your project:
+
+    ```bash
+    gcloud services enable --project $PROJECT_ID  container.googleapis.com                \
+                                                    gkehub.googleapis.com                 \
+                                                    anthosconfigmanagement.googleapis.com
+
+    gcloud alpha container hub config-management enable  --project $PROJECT_ID
+    ```
+
+1. Create cluster using terraform using defaults other than the project:
+
+    ```bash
+    # obtain user access credentials to use for Terraform commands
+    gcloud auth application-default login
+
+    # continue in /terraform directory
+    cd terraform
+
+    terraform init 
+    terraform plan -var=project=$PROJECT_ID
+    terraform apply -var=project=$PROJECT_ID
+    ```
+
+1. To verify things have sync'ed, you can use `gcloud` to check status:
+
+    ```bash
+    gcloud alpha container hub config-management status --project $PROJECT_ID
+    ```
+
+    In the output, notice that the `Status` will eventually show as `SYNCED` and the `Last_Synced_Token` will match the repo hash.
+
+1. To see wordpress itself, you can use the kubectl proxy to connect to the service:
+
+    ```bash
+    # get values from cluster that was created
+    export CLUSTER_ZONE=`echo google_container_cluster.cluster.location | terraform console | sed s/\"//g`
```

Comment:
We could just export this as an output in the example config and use `terraform output`

Reply:
Thanks. If I make Terraform output, I still need this one line export statement to set env variable, or env variable can be set automatically?

Reply:
You will still need to export var. Once you add an output like
```
output "location" {
  value = google_container_cluster.cluster.location
}
```
then it can be exported like `export CLUSTER_ZONE=$(terraform output -raw location)`

Reply:
ACK, will change.